src/main/resources/servers/broad-dev-jcarlton.json

```diff
@@ -7,5 +7,6 @@
   "samUri": "https://sam.dsde-dev.broadinstitute.org",
```

Comment:
This person's no longer on the team, this file can be deleted. :)

Same for wchamber